src/System.Security.Cryptography.Encoding/src/Configurations.props

```diff
@@ -3,8 +3,9 @@
   <PropertyGroup>
     <BuildConfigurations>
       net463-Windows_NT;
+      netstandard-OSX;
```

Comment:
Is this really unique to OSX independent of the Unix build? If so then this looks correct.